samples/Beginner/Beginner.sln

```diff
@@ -0,0 +1,80 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.12.35506.116
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{11A8DD76-328B-46DF-9F39-F559912D0360}") = "1-BlinkLed", "BlinkLed\1-BlinkLed.nfproj", "{558B53B9-75E0-4C13-AFF1-8614B3E0BAA6}"
+EndProject
+Project("{11A8DD76-328B-46DF-9F39-F559912D0360}") = "2-Button", "Button\2-Button.nfproj", "{3FA7A8F9-31FB-45B6-8727-D02C17D15BBA}"
+EndProject
+Project("{11A8DD76-328B-46DF-9F39-F559912D0360}") = "3-ButtonNuget", "ButtonNuget\3-ButtonNuget.nfproj", "{5B17A480-3C0F-4D73-B941-E9F8C2A07925}"
+EndProject
+Project("{11A8DD76-328B-46DF-9F39-F559912D0360}") = "5-Buzzer", "Buzzer\5-Buzzer.nfproj", "{9419FC61-403F-4E3D-BE93-B26D4A23B682}"
```

Comment:
Does this file affects order of projects in visual studio? If so, 5 and 4 should be switched.